lib/assessment.js

```diff
@@ -1,11 +1,19 @@
 'use strict';
 
+// I don't feel great about this!
+
 // Question 1
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
-
+  var minValue = arguments[0];
+  for (var i = 1; i < arguments.length; i++) {
+    if (arguments[i] < minValue) {
+      minValue = arguments[1];
```

Comment:
That `1` should be an `i`, but otherwise this looks right.

Here's the concept. How do we know who the fastest person in the world is? We don't race every potential runner against each other. Instead, we have a record; the fastest person is the one who beats that record.

This principle can be used to find the __est element of any set, be it largest, smallest, or any other single-value score.